### DISCUSSION TITLE
Mobile UI is severely laggy

### DISCUSSION THREAD

**Discussion by ChiefBroseph** [lllyasviel/stable-diffusion-webui-forge#1736] [Q&A]
I connect to my computer from my phone, and interacting with the UI is very sluggish, especially when editing text fields. I've used A1111 since the beginning without this issue popping up. I have listen, insecure extension access, and no gradio queue commands enabled. It doesn't run slow when using the desktop browser. Any hidden settings that might be causing this?

**Comment by patrickchamp** [comment]
do you have a auto tag fill extension enabled?

**Comment by kevinnio** [comment]
I was having a similar issue and just noticed that many components in Forge's UI either throw errors or trigger warnings in the browser's console. I can also see a spikes in CPU usage while doing simple interactions, like dragging around a slider for example. As a web developer this makes me believe the UI needs a good cleanup. Comparing to other A1111-based UIs, they don't seem to suffer the same issues. reForge for example runs perflectly fine.

**Comment by Vagrin** [comment]
UI hyper sluggish for me as well since last update or two. On a PC.
